.github/workflows/prbot.yml

```
@@ -16,6 +16,13 @@ jobs:
           node-version-file: '.nvmrc'
       - run: npm ci
       - run: npm run pr:report
+  lockfile-version:
+    name: 'Check lockfile version'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+      - run: node scripts/check-lockfile-versions.js $(find ./ -xdev -wholename '**/package-lock.json')
```

might be easier to have a bash script than add a new separate js script.
example:
```suggestion
      - run: |
        exitCode=0
        for lockfile in $(git ls-files {**/,}package-lock.json); do
          version=$(cat $lockfile | jq .lockfileVersion)
          if [ $version -ne 3 ]; then
            echo "Expected lockfileVersion of $lockfile to be 3." >&2
            exitCode=1
          fi
        done
        exit $exitCode
```

---

Also, we should probably use ls-files instead of find since`find` searches through `node_modules`